src/briefcase/integrations/docker.py

```diff
@@ -904,7 +907,9 @@ def prepare(
                     self.tools.subprocess.run(
                         [
                             "docker",
+                            "buildx",
                             "build",
+                            "--load",
                             "--progress",
```

Comment:
Added `buildx` so it is explicit that `buildx` is building the image. Added `--load` because it is only implied when using the default build container; if you use a custom one, the image isn't loaded in to the local Docker image cache.